examples/advanced/filters/package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "style-dictionary-filters",
+  "version": "1.0.0",
+  "description": "",
+  "main": "build/index.js",
+  "files": [
+    "build",
+    "properties"
+  ],
+  "scripts": {
+    "build": "node ./build.js",
```

Comment:
If we remove the `build.js` file we can change this to `"build": "style-dictionary build"`

Reply:
note necessary